### PR TITLE
Datapicker steps improvments

### PR DESCRIPTION
**Pull request by mfka**

### CODE REVIEW

**Comment by mfka** [comment]
Changes:
- Move steps to 
- block next step if file is not selected
- generate more generic client
- @szymonpoltorakrst move driver picker outside the component

File: src/components/DrivePicker.tsx [hunk 1]
```diff
@@ -1,97 +1,42 @@
...
 import StorageIcon from '@material-ui/icons/Storage';
-import React, { useContext, useState } from 'react';
+import React, { useContext } from 'react';
 import GooglePicker from 'react-google-picker';
 import { SpreadsheetCtx } from 'src/contexts/spreadsheet.context';
-import { UserCtx } from 'src/contexts/user.context';
-import { IUser } from 'src/types';
-import { ISpreadsheet } from 'src/types/spreadsheet';
-
+import { StepCtx } from 'src/contexts/step.context';
```
Comment (poltorak) [inline src/components/DrivePicker.tsx]: I'm not sure about such imports, this `src/` smells

File: src/components/Stepper/Steps.tsx [hunk 2]
```diff
@@ -42,7 +33,7 @@ const Steps = () => {
           <Stepper
             alternativeLabel={true}
             nonLinear={true}
-            activeStep={activeStep}
+            activeStep={(activeStep && activeStep.number) || undefined}
```
Comment (poltorak) [inline src/components/Stepper/Steps.tsx]: You shouldn't assign `undefined`. Undefined means that is was never defined, please use `null`

File: src/components/DrivePicker.tsx (hunk 1 above)
Comment (mfka) [reply src/components/DrivePicker.tsx]: It is default react root directory “Absolute Imports with Create React App” by Kyle Truong https://link.medium.com/witlKyPZKU

File: src/components/Stepper/Steps.tsx (hunk 2 above)
Comment (mfka) [reply src/components/Stepper/Steps.tsx]: Stepper component from material ui is accepting number or undefined . What in this case?

File: src/components/DrivePicker.tsx (hunk 1 above)
Comment (poltorak) [reply src/components/DrivePicker.tsx]: ok

File: src/components/Stepper/Steps.tsx (hunk 2 above)
Comment (poltorak) [reply src/components/Stepper/Steps.tsx]: Well then use undefined :)

Comment (pawlakzn) [reply src/components/Stepper/Steps.tsx]: Or the first one as the default?

**Pull request merged**